navit/osd/core/osd_core.c

```diff
@@ -2239,7 +2239,10 @@ static void osd_nav_toggle_announcer_draw(struct osd_priv_common *opc, struct na
         gr_image = graphics_image_new_scaled(opc->osd_item.gr, path, opc->osd_item.w, opc->osd_item.h);
         if (!gr_image) {
             g_free(path);
-            path = graphics_icon_path("unknown.png");
+            if (this->active)
+                path = graphics_icon_path("sound_on.png");
+            else
+                path = graphics_icon_path("sound_off.png");
```

Comment:
Strictly speaking this dosn't belong into this PR

Reply:
You are right, do I need to remove it?